compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp

```diff
@@ -55,10 +55,8 @@ static int64_t getLinearizedSize(SmallVector<int64_t> &shape) {
 
 /// Translate the memory space into shared / local.
 static FailureOr<std::string> translateMemorySpace(IntegerAttr memorySpace) {
-  if (isConstantIntValue(memorySpace, 1))
-    return std::string("shared");
-  if (isConstantIntValue(memorySpace, 2))
-    return std::string("local");
+  if (isConstantIntValue(memorySpace, 1)) return std::string("shared");
+  if (isConstantIntValue(memorySpace, 2)) return std::string("local");
```

Comment:
I wonder why the `clang-format` didn't catch this as an error while checking formatting.

Reply:
Actually, where does CI run clang-format?